java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java

```diff
@@ -57,62 +62,278 @@ public TaskManagerClient(String endpoint) {
+        TaskManager.RunBatchSqlRequest request = TaskManager.RunBatchSqlRequest.newBuilder()
+                .setSql(sql)
+                .setOutputPath(output_path)
+                .setDefaultDb(default_db)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.RunBatchSql(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getMsg();
+    }
+    /**
+     * RunBatchAndShow.
+     *
+     * @param sql
+     *
+     * @throws Exception
+     */
+    public String runBatchAndShow(String sql) throws Exception {
+        TaskManager.RunBatchAndShowRequest request = TaskManager.RunBatchAndShowRequest.newBuilder()
+                .setSql(sql)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.RunBatchAndShow(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getMsg();
+    }
+    /**
+     * RunBatchAndShow.
+     *
+     * @param sql,default_db(optional) .
+     *
+     * @throws Exception
+     */
+    public String runBatchAndShow(String sql,String default_db) throws Exception {
+        TaskManager.RunBatchAndShowRequest request = TaskManager.RunBatchAndShowRequest.newBuilder()
+                .setSql(sql)
+                .setDefaultDb(default_db)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.RunBatchAndShow(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getMsg();
+    }
+    /**
+     * Show the job rely on id.
+     *
+     * @param id of job.
+     * @return the JobInfo object.
+     * @throws Exception
+     */
+    public TaskManager.JobInfo showOneJob(int id) throws Exception {
+        TaskManager.ShowJobRequest request = TaskManager.ShowJobRequest.newBuilder()
+                .setId(id)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.ShowJob(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getJob();
+    }
+    /**
+     * ImportOnlineData
+     *
+     * @param sql .
+     *
+     * @throws Exception
+     */
+    public String importOnlineData(String sql) throws Exception {
```

Comment:
Reuse the method below `importOnlineData(String sql,String default_db)` by passing the parameters instead of copying the similar logic again.